tests/unittest_brain.py

```diff
@@ -1469,6 +1469,19 @@ def test_typing_types(self):
             inferred = next(node.infer())
             self.assertIsInstance(inferred, nodes.ClassDef, node.as_string())
 
+    @test_utils.require_version(minver="3.7")
+    def test_plain_tuple_type(self):
```

Comment:
I can't find a relation between this test and your modification. Can you explain a little bit please?

Reply:
Before I added the cache invalidation logic in `TransformVisitor` the `test_tuple_type` test was the only test that failed. I added this test while I was debugging that, but I guess it doesn't make any sense without an explanation.